lib/services/api_path.dart

```diff
@@ -0,0 +1,35 @@
+class APIPath {
+  // static apikey() => "20562BC7-5687-43F1-A7C1-1B61D5037508";
```

Comment:
Nice, although we don't require API key yet, we might add it in the future. Always prepared. 👍